django/db/models/sql/query.py

```diff
@@ -46,6 +46,8 @@
 
 
 def get_field_names_from_opts(opts):
+    if opts is None:
+        return set()
```

Comment:
This is only tested in `constraints.tests.CheckConstraintTests.test_validate` and `constraints.tests.UniqueConstraintTests.test_validate_expression_condition`. Can we add a regression test in the `FromEmptyQuery`?